src/libs/scroll-to-top-on-collapse.js

```diff
@@ -0,0 +1,22 @@
+export default () => {
+	console.log('hihi')
+	const btns = document.querySelectorAll('.file-actions > .js-details-target')
+	const prToolbar = document.querySelector('.pr-toolbar')
+	const toolbarHeight = prToolbar.getBoundingClientRect().height
+
+	btns.forEach((b) => {
+		// get the diff container for this button
+		const parent = b.parentElement.parentElement.parentElement
+
+		b.addEventListener('click', () => {
```

Comment:
Can we avoid the loop and use a delegate listener instead? Big PRs are loaded via ajax. Something like this instead of approximately the first 6 lines of this file:

```js
$('.js-diff-progressive-container').on('click', '.js-details-target', fn);
```